Automattic/wp-calypso
# Skeleton of a reblogging flow

Related to https://github.com/Automattic/wp-calypso/issues/85458

## Proposed Changes

* Adds a new reblog flow to handle the scenario where users without a site click on the post reblog button and attempt to create a site. The "create a site" button will be updated to navigate to a simple reblog flow that will navigate the user to the post editor with the post embed on completion of the flow.

<img width="640" alt="CleanShot 2023-12-29 at 12 00 02@2x" src="https://github.com/Automattic/wp-calypso/assets/10482592/6909dbdd-4973-47ad-b7f2-e6cbf607442d">


## Testing Instructions
 1. Incognito
 2. Visit [this link](http://calypso.localhost:3000/post?url=https%3A%2F%2Fmmm434.wordpress.com%2F2023%2F09%2F26%2Fhello-world%2F%3Fpage_id%3D3&is_post_share=true&v=5)
 3. You should see the button to start the reblogging flow
